src/utility/util.js

```diff
@@ -0,0 +1,33 @@
+const { text } = require("express");
+const fs = require("fs");
+const path = require("path");
+
+const getNotesList = () => {
+  // get manga data fom the db file by reading, then parse the string and re.json
+
+  const notesJSON = fs.readFileSync(
+    path.join(__dirname, "../../db/db.json"),
+    "utf-8"
+  );
+
+  const notes = JSON.parse(notesJSON);
+
```

Comment:
No need to create and store a variable here just to simply return it. We can just directly return this statement:

`return JSON.parse(jsondata)`